pages/team.js

```diff
@@ -267,7 +306,39 @@ and participating in academic competitions with his friends.`}
               />
             </Grid>
             <br />
-            <Text variant="title" color="orange" sx={{ lineHeight: '1.75em' }}>
+            <Text
+              variant="title"
+              color="orange"
+              sx={{ lineHeight: '1.75em', fontSize: ['42px', 6] }}
+            >
+              Bots
+            </Text>
+            <Grid columns={[1, null, 2]} gap={4}>
+              <Bio
+                name="Scrappy"
+                teamRole="Book Keeper"
+                text="The bot that eats trash (and manages #scrapbook)."
+                img="/team/scrappy.png"
+              />
+              <Bio
+                name="Orpheus"
+                teamRole="Assistant"
+                text="The bot that helps the team manage everything."
+                img="/team/orpheus.png"
+              />
+              <Bio
+                name="Toriel"
+                teamRole="Bouncer"
```

Comment:
I'm not sure that Bouncer is appropriate here? It sort of has a strange connotation. Thoughts on changing it to "Greeter"?

Reply:
Yes i agree, greeter sounds good! Thanks :))